### DISCUSSION TITLE
Since a few days A1111 is not working properly

### DISCUSSION THREAD

**Discussion by darth2602** [TheLastBen/fast-stable-diffusion#1111] [Q&A]
Hello

For some time now (since the arrival of version 2.x) A1111 has been working much less well at home (in colab) with the huggingface trained models.  I regularly have a "blocking" (?)  (1) I launch a render, which works, 2) I press Generate again and it does not start) and so regularly I have to leave the interface and restart the Start stable-diffusion step ...

I specify that I am in version 1.5, I do not use the 2.0 / 2.1 I have already deleted the SD folder and I have loaded only the 1.5 models
  
any ideas ?

**Comment by TheLastBen** [comment]
I switched to the gradio server, are you using it ?

**Comment by darth2602** [comment]
ho....no....   on est obligé d'utiliser gradio ?   ok ...

**Reply by TheLastBen** [reply]
C'est le cas depuis presque un mois

**Reply by darth2602** [reply]
tu parle français ?

**Reply by darth2602** [reply]
(ha, oui machinalement j'ai écrit ma dernière réponse en français sans traduire....)

**Reply by darth2602** [reply]
Du coup en cas de problème j'écris en Français (ma langue maternelle) ou en anglais ?

**Reply by TheLastBen** [reply]
Oui, si tu veux écrire en français, ce n'est pas grave.
